### PR TITLE
Display allowance config.

### PR DESCRIPTION
**Pull request by patjanawat**
- Display the allowance of each accounts based on the store (**Closes**)

closes https://github.com/mesg-foundation/atd/issues/49

### CODE REVIEW

File: frontend/components/admin/allowance.vue [hunk 1]
```diff
@@ -34,14 +34,14 @@
...
-    <v-data-table :headers="headers" :items="allowances" class="elevation-1">
+    <v-data-table :headers="headers" :items="allowances">
       <template v-slot:no-data>
         <v-alert :value="true" color="error" icon="warning">Sorry, nothing to display here :(</v-alert>
       </template>
       <template v-slot:items="props">
         <td>{{ props.item.name }}</td>
         <td class="text-xs-left">{{ props.item.address }}</td>
-        <td class="text-xs-left">{{ props.item.value }}</td>
+        <td class="text-xs-left">{{ props.item.allowance }}</td>
```
Comment (NicolasMahe) [inline frontend/components/admin/allowance.vue]: Could you format the allowance value the same way as all number in MESG token?
xxx,xxx.xx

File: frontend/components/admin/allowance.vue [hunk 2]
```diff
@@ -34,14 +34,14 @@
       </v-card>
     </v-dialog>
 
-    <v-data-table :headers="headers" :items="allowances" class="elevation-1">
+    <v-data-table :headers="headers" :items="allowances">
```
Comment (NicolasMahe) [inline frontend/components/admin/allowance.vue]: Could you remove the pagination? there is only 4 allowances.

File: frontend/components/admin/allowance.vue (hunk 1 above)
Comment (patjanawat) [reply frontend/components/admin/allowance.vue]: Done.

File: frontend/components/admin/allowance.vue (hunk 2 above)
Comment (patjanawat) [reply frontend/components/admin/allowance.vue]: Done.

**Pull request merged**